rtCamp/web-stories-wp
# Replaces custom Lightbox with `amp-lightbox` on AMP pages.

## Summary

This PR updates the story renderer to use `amp-lightbox` instead of a custom lightbox solution.

## Relevant Technical Choices

To bind individual stories we are adding one `amp-lightbox` for each story rendered.

## To-do

- test with `amp-script` if wrapping the elements and adding the script that is in place for the non-AMP pages for
loading the story in a lightbox.

## User-facing changes

- n/a

## Testing Instructions

- See 'web stories list' block on AMP pages.
- Check that now we have multiple `amp-lightbox` nodes.

---
